### PR TITLE
[Rule Tuning] Kerberos Cached Credentials Dumping

### LINKED ISSUE #2102: [Rule Tuning] Kerberos Cached Credentials Dumping
## Link to rule
https://github.com/elastic/detection-rules/blob/main/rules/macos/credential_access_kerberosdump_kcc.toml

## Description

- Updated description to include threat actor utilization

### PR DESCRIPTION
**Pull request by Mikaayenson**
## Issues

Resolves #2102 

## Summary
- Updated description to include threat actor utilization

### CODE REVIEW

File: rules/macos/credential_access_kerberosdump_kcc.toml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2022/07/13"
 
 [rule]
 author = ["Elastic"]
-description = "Identifies the use of the Kerberos credential cache (kcc) utility to dump locally cached Kerberos tickets."
+description = """Identifies the use of the Kerberos credential cache (kcc) utility to dump locally cached Kerberos
+tickets. Adversaries may attempt to dump credentials to obtain account login information in the form of a ticket.
+These tickets can be leveraged for lateral movement."""
```
Comment (w0rk3r) [inline rules/macos/credential_access_kerberosdump_kcc.toml:10]:
```suggestion
description = """
Identifies the use of the Kerberos credential cache (kcc) utility to dump locally cached Kerberos tickets. Adversaries
may attempt to dump credential material in the form of tickets that can be leveraged for lateral movement.
"""
```

**Review by w0rk3r — APPROVED** [review]
Left a optional suggestion. LGTM

**Review by Samirbous — APPROVED** [review]
bump up the update date.

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**